src/components/ProductList/cart.js

```diff
@@ -0,0 +1,35 @@
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button';
+import StyledProductCard from '../ProductCard';
+
+import StyledProductList from './styled';
+
+function ProductListCart() {
+	const cartProducts = useStore(state => state.cartProducts);
+	const deleteFromCart = useStore(state => state.deleteFromCart);
+
+	return (
+		<>
+			<StyledProductList>
+				{cartProducts.map(cartProduct => {
+					return (
+						<StyledProductCard key={cartProduct.id}>
+							<p>{cartProduct.name}</p>
+							<p>{cartProduct.price}</p>
+							<StyledButton
+								variant="delete"
+								onClick={() => {
+									deleteFromCart(cartProduct.id);
+								}}
+							>
+								Delete
+							</StyledButton>
+						</StyledProductCard>
+					);
+				})}
+			</StyledProductList>
+		</>
```

Comment:
You don't need the fractions here as the `StyledProductList` already is the container element here.
```suggestion
			<StyledProductList>
				{cartProducts.map(cartProduct => {
					return (
						<StyledProductCard key={cartProduct.id}>
							<p>{cartProduct.name}</p>
							<p>{cartProduct.price}</p>
							<StyledButton
								variant="delete"
								onClick={() => {
									deleteFromCart(cartProduct.id);
								}}
							>
								Delete
							</StyledButton>
						</StyledProductCard>
					);
				})}
			</StyledProductList>
```